docs/source/getting_started.rst

```diff
@@ -0,0 +1,632 @@
+
+Overriding keys directly
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now, imagine that in addition to this you want to take one of the linear layer's weights from a third model overriding 
+the weights from the first checkpoint's head. Let the weights from the third model be:
+
+.. code-block::
+  :caption: ./head3_linear2/checkpoint/path.ckpt
+
+  linear.2: torch.tensor(10)
+
+In this case, your configuration should look like this:
+
+.. code-block:: yaml
+
+  task:
+  # ...
+    load_checkpoint:
+      base_ckpt_path: ./task/checkpoint/path.ckpt
+      overridden_name2ckpt_path:
+        head: ./head2/checkpoint/path.ckpt
+        head.linear.2: ./head3_linear2/checkpoint/path.ckpt
+      strict: true
+
+And the total model weights will be (a deeper specified key overrides a shallower specified key):
+
+.. code-block::
+
+  backbone.linear.1: torch.tensor(1),
+  backbone.linear.2: torch.tensor(1),
+  head.linear.1: torch.tensor(3),
+  head.linear.2: torch.tensor(10),
+  head.linear.3: torch.tensor(7)
+
+Excluding keys
+^^^^^^^^^^^^^^
+
+Let's now see how will your config look like if you decide to leave one of the layers with default initialization:
+
+.. code-block:: yaml
+
+  task:
+  # ...
+    load_checkpoint:
+      base_ckpt_path: ./task/checkpoint/path.ckpt
+      overridden_name2ckpt_path:
+        head: ./head2/checkpoint/path.ckpt
+        head.linear.2: ./head3_linear2/checkpoint/path.ckpt
+      exclude_keys:
+        head.linear.3
+      strict: false
+
+In this case, `strict: false` allows us to load model weights partially skipping the `head.linear.3` layer.
+
+After loading, your total model weights will be:
+
+.. code-block::
+
+  backbone.linear.1: torch.tensor(1),
+  backbone.linear.2: torch.tensor(1),
+  head.linear.1: torch.tensor(3),
+  head.linear.2: torch.tensor(10),
+  head.linear.3: torch.tensor(0)
+
+Absolute model keys
+^^^^^^^^^^^^^^^^^^^
+
+If one of your checkpoints has full length keys (we call them *absolute* keys), you don't need to convert the keys 
+to *relative* ones, TorchOk will read them as having an overridden key prefix included. Let your second model's head's 
+weights be:
+
+.. code-block::
+  :caption: ./head2/checkpoint/path.ckpt (absolute keys)
+
+  head.linear.1: torch.tensor(3),
+  head.linear.2: torch.tensor(5),
+  head.linear.3: torch.tensor(7)
+
+The resulting merged checkpoints won't change from the above examples.
+
+Skip loss calculation on validation
+===================================
+
+Sometimes you don't need to calculate loss during the validation. If you don't override `validation_step` of the 
+`BaseTask` you can set `compute_loss_on_valid: false` (default is `true`) to skip loss calculation on validation.
+
+Seeding randomizers
+===================
+
+To increase reproducibility of your experiments, you can provide seed values to internal randomizers. Usually, 
+Deep Learning Engineers configure seeds for all the randomizers. This can be achieved in TorchOk by specifying 
```

Comment:
To fullty reproducability we must set determenistic=True to Trainer https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#:~:text=To%20ensure%20full%20reproducibility%20from%20run%20to%20run%20you%20need%20to%20set%20seeds%20for%20pseudo%2Drandom%20generators%2C%20and%20set%20deterministic%20flag%20in%20Trainer.